addons/medical_treatment/CfgVehicles.hpp

```diff
@@ -29,6 +29,13 @@ class CfgVehicles {
                 statement = "";
                 icon = "\a3\ui_f\data\IGUI\Cfg\Actions\eject_ca.paa";
                 selection = "";
+                class GVAR(buryBodyBag) {
+                    displayName = CSTRING(DigGrave);
+                    distance = HANDCUFFS_DISTANCE;
+                    condition = QUOTE([_this#1] call FUNC(canDigGrave));
```

Comment:
Won't this need tweaking because of the isAwake check?

Reply:
I should've put the isAwake check here now that I think about it.

Reply:
Moved to the unit action in #9451